specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-03-01-preview/examples/AppComponent_CreateOrUpdateAppComponents_MaximumSet_Gen.json

```diff
@@ -0,0 +1,63 @@
+{
+  "parameters": {
+    "name": "_3w-",
+    "api-version": "2022-03-01-preview",
+    "body": {
+      "resourceId": "a",
+      "testId": "aaaaaaaaaaaaaaaaaaaaaaaaaaa",
+      "testRunId": "aaaaaaaaaaaaa",
+      "name": "aaaaaaaaaaaaaaaa",
+      "value": {
+        "key5132": {
```

Comment:
This is not an acceptable example. Examples should mirror, within reason, an actual request.